spyder/api/plugins.py

```diff
@@ -1606,10 +1611,10 @@ def __init__(self, parent, configuration):
         # Render all toolbars as a final separate step on the main window
         # in case some plugins want to extend a toolbar. Since the rendering
         # can only be done once!
-        widget.get_main_toolbar()._render()
-        for __, toolbars in widget._aux_toolbars.items():
-            for toolbar in toolbars:
-                toolbar._render()
+        widget._main_toolbar._render()
+        widget._corner_toolbar._render()
+        for __, toolbar in widget._auxiliary_toolbars.items():
```

Comment:
Should the widget attributes above (`_main_toolbar`, `_corner_toolbar` and `_auxiliary_toolbars`) be called using a `get_` method (since seems like they are private)?

Reply:
Yes, much better like that. Will fix.